Makefile

```diff
@@ -0,0 +1,108 @@
+# Makefile for easier installation and cleanup.
+#
+# Uses self-documenting macros from here:
+# http://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
+
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+MAKEFLAGS += --warn-undefined-variables --no-builtin-rules
+
+PACKAGE=readabilipy
+DOC_DIR=./docs
+VENV_DIR=/tmp/rdpy_venv
+TEST_DIR=./tests
+
+.PHONY: help
+
+.DEFAULT_GOAL := help
+
+help:
+	@grep -E '^[0-9a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) |\
+		 awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-15s\033[0m\
+		 %s\n", $$1, $$2}'
```

Comment:
What is this doing?

Reply:
Gives a little help message for each target when you type ``make`` on the command line without any argument:

```
$ make
install        	 Install for the current user using the default python command
release        	 Make a release
dist           	 Make Python source distribution
test           	 Run unit tests
doc            	 Build documentation with Sphinx
clean          	 Clean build dist and egg directories left after install
```